sklbench/datasets/README.md

```diff
@@ -2,8 +2,9 @@
 
 Data handling steps:
 1. Load data:
-   - If not cached: download/generate dataset and put it in raw and/or usual cache
+   - If not cached and not available at specified `DATASETSROOT`: download/generate dataset and put it in raw and/or usual cache
    - If cached: load from cached files
+   - If `DATASETSROOT` path to indicated file exists: load from specified CSV
```

Comment:
Why `DATASETSROOT` is needed if `raw_cache` and `cache` exist?

Reply:
With `DATASETSROOT` if the datasets exist there is no need to ever download the data from source (not even before the cache exists) - is this possible with the existing raw_cache and cache logic?